OpenTreeMap/otm-vagrant
# Update boostrap, init script, and settings to support celery 4+

## Overview

We updated otm-core to use Celery 4 in https://github.com/OpenTreeMap/otm-core/commit/1147a7e3 as part of PR https://github.com/OpenTreeMap/otm-core/pull/3184.

This commit updates the provisioning script, init script, and related settings to support the newer version.

- `celeryd` service replaced with `celery` which launches the standalone
  `celery` executable rather than a Django management command.
- `BROKER_URL` setting renamed to `CELERY_BROKER_URL`. The `otm-core` Celery
  setup requires that all configuration settings to have a `CELERY_` prefix.

Connects https://github.com/OpenTreeMap/otm-clients/issues/454

## Testing Instructions

 * Destroy any existing VM
 * `get-repos.sh`
 * `vagrant up`
 * Browse http://localhost:7070/accounts/register/ and create an account
 * Browse the account confirmation link returned from 
```
vagrant ssh -c "cat /usr/local/otm/emails/*"
``` 
 * Create an instance
```
vagrant ssh -c "/usr/local/otm/app/opentreemap/manage.py create_instance Philadelphia --url_name=philly --user={{YOUR_USERNAME}} --center=-75.1,40.0"
```
* Browse http://localhost:7070/philly/management/bulk-uploader/ and log in.
* Use the "Tree Import" section to upload any random, small-ish CSV and verify that it appears under "Active Tree Imports" and eventually moves to "Finished Tree Imports"
